lib/ast-utils/import-parse.js

```diff
@@ -67,15 +78,25 @@ function addNode(node, registry) {
 * @param  {[type]} htmlString A utf8, html5 document containing polymer element or module definitons.
 * @return {ParsedImport}
 */
-var importParse = function importParse(htmlString) {
+var importParse = function importParse(htmlString, href) {
   var doc;
   try {
-    doc = dom5.parse(htmlString);
+    doc = dom5.parse(htmlString, {locationInfo: true});
   } catch (err) {
     console.log(err);
     return null;
   }
 
+  // Add line/column information
+  dom5.treeMap(doc, function(node) {
+    if (node.__location && node.__location.start >= 0) {
+      node.__locationDetail = getLineAndColumn(htmlString, node.__location.start);
```

Comment:
If you know you're going to repeatedly call `getLineAndColumn` for the same document, you could speed it up by caching the line lengths and line start offsets.